love/src/components/Watcher/AlarmAudio/AlarmAudio.jsx

```diff
@@ -86,13 +86,17 @@ export default class AlarmAudio extends Component {
     });
     this.newCriticalSound = new Howl({
       src: [newCriticalFile],
-      loop: 1,
       onplayerror: () => {
         console.error('Error playing sound for critical alarm: ', newCriticalFile);
       },
       onloaderror: () => {
         console.error('Error loading sound for critical alarm: ', newCriticalFile);
       },
+      onend: () => {
+        setTimeout(() => {
+          this.newCriticalSound.play();
+        }, ALARM_SOUND_THROTLING_TIME - 3000); // subtract alarm sound length
```

Comment:
The value of 3000 (alarm sound length) was repeated several times in this file. Please extract to a constant and use that everywhere.